metagraph/tests/annotation/taxonomy/test_taxonomy.cpp

```diff
@@ -0,0 +1,91 @@
+#include "gtest/gtest.h"
+
+#define TESTING
+
+#include <tsl/hopscotch_map.h>
+#include <tsl/hopscotch_set.h>
+
+#include "annotation/taxonomy/tax_classifier.hpp"
+
+namespace mtg {
+namespace test {
```

Comment:
don't we use unnamed namespace for tests?
```suggestion
namespace {
```